lib/reporters/base.js

```diff
@@ -188,18 +193,21 @@ function stringifyDiffObjs(err) {
  * @param {string} expected
  * @return {string} Diff
  */
+
 var generateDiff = (exports.generateDiff = function (actual, expected) {
   try {
-    const diffSize = 2048;
-    if (actual.length > diffSize) {
-      actual = actual.substring(0, diffSize) + ' ... Lines skipped';
-    }
-    if (expected.length > diffSize) {
-      expected = expected.substring(0, diffSize) + ' ... Lines skipped';
-    }
-    return exports.inlineDiffs
+    const maxLen = exports.maxDiffSize || Number.MAX_SAFE_INTEGER;
```

Comment:
`Number.MAX_SAFE_INTEGER` is not IE11 compatible. Our (week) browser tests have passed, nevertheless I prefer to replace it, since `generateDiff()` is part of our browser bundle. You can insert a condition `if (exports.maxDiffSize > 0) { ...` otherwise don't slice anything. With this solution you also prevent weird results when `maxDiffSize` is a negativ number.